challenge.js

```diff
@@ -26,6 +27,7 @@
  */
 
 // ANSWER
+A is Es ancestor.  E may inherit properties from A, unless D has overwritten them.
```

Comment:
This answer shows strength in CSS inheritance. You're right that A is E's ancestor, and that E is A's descendant. This question was looking for a more specific description of their relationship on the DOM. 3/4